default.nix

```diff
@@ -806,7 +806,7 @@ rec {
           nixpkgs.rlwrap # for `rlwrap moc`
           nixpkgs.difftastic
           nixpkgs.openjdk nixpkgs.z3 nixpkgs.jq # for viper dev
-        ]
+        ] ++ nixpkgs.lib.optional stdenv.isDarwin nixpkgs.darwin.apple_sdk.frameworks.Security
```

Comment:
This was in my checkout too. It helps linking `ic-repl` on the Mac, so it is actually useful. But I can back it out if you think it doesn't belong here.